packages/@aws-cdk/aws-appmesh/test/integ.mesh.ts

```diff
@@ -67,15 +67,13 @@ router.addRoute('route-1', {
 const node2 = mesh.addVirtualNode('node2', {
   serviceDiscovery: appmesh.ServiceDiscovery.dns(`node2.${namespace.namespaceName}`),
   listeners: [appmesh.VirtualNodeListener.http({
-    healthCheck: {
+    healthCheck: appmesh.HealthCheck.http({
       healthyThreshold: 3,
       interval: cdk.Duration.seconds(5),
       path: '/check-path2',
-      port: 8080,
-      protocol: appmesh.Protocol.HTTP,
```

Comment:
Why these changes? We really should strive to keep changes to tests to a minimum when doing these kinds of refactorings.